sql/hive/src/test/resources/golden/timestamp cast #3-0-732ed232ac592c5e7f7c913a88874fd2

```diff
@@ -1 +0,0 @@
-1.2
```

Comment:
sorry I was wrong. calling `TIMESTAMP_SECONDS` and casting double to int is not the same as the legacy cast double to timestamp.

Can we simply remove that test?

Reply:
if we remove it, there will be lack of testing for casting from double to timestamp for createQueryTest, we may need to add the legacy to let the test case pass.

Reply:
It's OK. We have tests for casting double to timestamp in `CastSuite`.

Reply:
BTW shall we remove this golden file since the test is removed?

Reply:
Ah.i have removed it this early morning just fews minutes after your comment.  it has disappeared in last commit.and the following test also has been finished